watertap/unit_models/mvc/components/tests/test_evaporator.py

```diff
@@ -29,6 +30,7 @@
 solver = get_solver()
 
 # -----------------------------------------------------------------------------
+@pytest.mark.skipif(sys.platform.startswith("darwin"), reason="Evaporator needs complied IDAES extensions")
```

Comment:
Just FYI, the predefined `@pytest.mark.requires_idaes_solver` marker can be used for this functionality (skipping/xfailing tests if IDAES extensions are not available).